modules/local/software/cellranger/mkgtf/main.nf

```diff
@@ -0,0 +1,35 @@
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process CELLRANGER_MKGTF {
+
+    label 'process_low'
+
+    publishDir "${params.outdir}",
+        mode: 'copy',
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
+
+    container "streitlab/custom-nf-modules-cellranger:latest"
+
+    input:
+        path(gtf)
+
+    output:
+        path("*.gtf")
```

Comment:
same here, software version :) 
also probably a good idea to add a 'emit' statement